cinema/models.py

```diff
@@ -42,6 +46,15 @@ class Movie(models.Model):
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
 
+    def movie_image_path(self: "Movie", filename: str) -> pathlib.Path:
+        filename = pathlib.Path(
+            f"{slugify(self.title)}-{uuid.uuid4()}"
+            + pathlib.Path(filename).suffix
+        )
+        return pathlib.Path("movies/") / filename
```

Comment:
The best practice is to move it out of this class so it's reusable by other models

Reply:
Got it